pycid/core/macid.py

```diff
@@ -24,7 +23,7 @@ class MACID(MACIDBase):
     def get_ne(self, solver: Optional[str] = "enumpure") -> List[List[StochasticFunctionCPD]]:
         """
         Return a list of Nash equilbiria in the MACID. By default, this finds all pure NE using the 'enumpure'
```

Comment:
shouldn't this default to the same behaviour as we discussed for SPE - i.e., in two player games, we want it to default into using enummixed so it finds all NE, and in N-player games we want it to default to enumpure unless no NE are found => it finds a mixed NE using other solver